src/main/flight/pid.c

```diff
@@ -179,6 +179,13 @@ void resetPidProfile(pidProfile_t *pidProfile)
         .motor_output_limit = 100,
         .auto_profile_cell_count = AUTO_PROFILE_CELL_COUNT_STAY,
         .horizonTransition = 0,
+        .airmode_min_slow_authority = 50,
+        .airmode_min_fast_authority = 50,
+        .airmode_med_slow_authority = 100,
+        .airmode_med_fast_authority = 100,
+        .airmode_max_slow_authority = 100,
+        .airmode_max_fast_authority = 100,
```

Comment:
is there really a need to make them all configurable?

Reply:
No but it was a nice way to get to test out code easily without needing to ever refresh. Updates to this code found in the "different mixer" branch of the emuflight code.

Reply:
Thanks for pointing that one out.